src/jdk.jdwp.agent/windows/native/libdt_socket/socket_md.c

```diff
@@ -266,9 +266,9 @@ dbgsysNetworkToHostLong(uint32_t netlong) {
 }
 
 /*
- * Below Adapted from PlainSocketImpl.c, win32 version 1.18. Changed exception
- * throws to returns of SYS_ERR; we should improve the error codes
- * eventually. Changed java objects to values the debugger back end can
+ * Below Adapted from PlainSocketImpl.c (removed in JDK18), win32 version 1.18.
```

Comment:
I think we'll need decide whether to keep this comment or not, I would be tempted to re-write it.

Reply:
I've removed the comment after discussing this offline. See 49125e7